Automattic/wp-calypso
# State: Move themes action types out

This PR moves the theme action types out of the monolithic file with action types. This won't bring immediate results in reducing size of main entry point, because there are still other pieces of state that are using these action types (sites, UI, notices), but once those are also modularized, the effect will be even bigger!

#### Changes proposed in this Pull Request

* State: Move themes action types out

#### Testing instructions

* Verify that the "themes" functionality works well both when logged in and logged out.
